flow/core/params.py

```diff
@@ -897,14 +917,71 @@ class SumoLaneChangeParams:
     ----------
     lane_change_mode : str or int, optional
         may be one of the following:
+        * "no_lc_safe" (default): Disable all autonomous changing but still
+          handle safety checks (collision avoidance and safety-gap enforcement)
+          in the simulation. Binary is [001000000000]
```

Comment:
We need to be careful about the word autonomous, this implies to me that the AVs cannot lane change. I think it's worth expanding the text here to be clearer about the effects.

Reply:
I think if you say "disable all SUMO lane changing" that might be good